markerclusterer/src/markerclusterer.js

```diff
@@ -124,6 +127,18 @@ function MarkerClusterer(map, opt_markers, opt_options) {
   this.imagePath_ = options['imagePath'] ||
       this.MARKER_CLUSTER_IMAGE_PATH_;
 
+  /**
+   * @type {Array}
+   * @private
+   */
+  this.imageList_ = ('imageList' in options && options['imageList'].length) ? [].concat(options['imageList']) : null;
```

Comment:
Why not have `imageList_` get initialized here with the default 1 to 5 images if it isn't provided, then below there'd be no need for different logic based on whether `imageList_` was specified. 

Also the check should be something like `this.imageList_ = options['imageList']  ? options['imageList'] : []` which is simpler and more compatible with older browsers.